Gruntfile.js

```diff
@@ -157,14 +157,21 @@ module.exports = function (grunt) {
           open: true,
           livereload: false,
           middleware: function (connect) {
+            var rewriteRules = [
+              '^/styles/extensions\.css$ /' + contextRoot + '/extensions/extensions.css [R]',
+              '^/scripts/extensions\.js$ /' + contextRoot + '/extensions/extensions.js [R]',
+              '^/$ /' + contextRoot + '/ [R=302]',
+              '^/' + contextRoot + '(.*)$ $1',
+              '!^/(config.js|(bower_components|scripts|images|styles|views|extensions)(/.*)?)$ /index.html [L]'
+            ];
+
+            // If config.local.js exists, use that instead of config.js.
+            if (grunt.file.exists('app/config.local.js')) {
+              rewriteRules.unshift('^/config.js$ /config.local.js [L]');
```

Comment:
No, we copy the `config.local.js` to `.tmp` for `grunt serve`, but I don't think that works here.

https://github.com/openshift/origin-web-console/blob/master/Gruntfile.js#L552-L563

I suspect copying has the same problem when you delete the file.